src/Sylius/Behat/Context/Api/Admin/ManagingShipmentsContext.php

```diff
@@ -61,13 +86,29 @@ public function iShouldSeeCountShipmentsInList(int $count = 1): void
         Assert::same($this->responseChecker->countCollectionItems($this->client->getLastResponse()), $count);
     }
 
+    /**
+     * @When I ship the shipment of order :order
+     */
+    public function iShipShipmentOfOrder(OrderInterface $order): void
+    {
+        $this->client->applyTransition((string) $order->getShipments()->first()->getId(), ShipmentTransitions::TRANSITION_SHIP);
+    }
+
+    /**
+     * @Then I should be notified that the shipment has been successfully shipped
+     */
+    public function iShouldBeNotifiedThatTheShipmentHasBeenSuccessfullyShipped(): void
+    {
+        Assert::same($this->responseChecker->getValue($this->client->getLastResponse(), 'state'), 'shipped', 'Shipment is not shipped');
+    }
+
     /**
      * @Then /^I should see the shipment of (order "[^"]+") as "([^"]+)"$/
      */
     public function iShouldSeeTheShipmentOfOrderAs(OrderInterface $order, string $shippingState): void
     {
         Assert::true(
-            $this->responseChecker->hasItemWithValues($this->client->getLastResponse(), [
+            $this->responseChecker->hasItemWithValues($this->client->index(), [
```

Comment:
Is it necessary change?

Reply:
yes, because i check shipment state from shipment index, no from lest response (which is the response after shipment